projects/extension/tests/vectorizer/test_vectorizer.py

```diff
@@ -1815,3 +1815,67 @@ def test_create_vectorizer_privs():
             , grant_to=>null
             );
             """)
+
+
+def test_weird_primary_key():
```

Comment:
WDYT about adding some context to this test? like the reason of being, why we are testing this change, etc. I know we always have github, but it will play in favour of developers when reading this test.